ref/netstandard1.3/Microsoft.Build/Microsoft.Build.cs

```diff
@@ -1337,6 +1342,12 @@ public sealed partial class ProjectTaskOutputPropertyInstance : Microsoft.Build.
         public string TaskParameter { get { throw null; } }
         public override Microsoft.Build.Construction.ElementLocation TaskParameterLocation { get { throw null; } }
     }
+    public partial class RequestedProjectState
+    {
+        public RequestedProjectState() { }
+        public System.Collections.Generic.IDictionary<string, System.Collections.Generic.List<string>> ItemFilters { get { throw null; } set { } }
+        public System.Collections.Generic.List<string> PropertyFilters { get { throw null; } set { } }
```

Comment:
How do we feel about this interface? Would it be preferable to have constructor arguments, or another data type? One thing of concern is that it'd be nice to enforce case-insensitivity on in-proc cases.